config/sync/user.role.mediator.yml

```diff
@@ -100,11 +100,14 @@ permissions:
   - 'update media'
   - 'use text format basic'
   - 'use text format limited'
+  - 'view all media revisions'
   - 'view any unpublished content'
   - 'view own unpublished content'
   - 'view own unpublished media'
   - 'view scheduled content'
   - 'view scheduled eventseries'
+  - 'view scheduled media'
+  - 'view scheduled taxonomy_term'
```

Comment:
Do you have any comments about why these should be added?

Reply:
Because it matches other permissions, and i think they're missing by mistake